### PR TITLE
Remove Custom Footer; Indicate No More Messages To Load

### PR DESCRIPTION
**Pull request by RoyalSix**
## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> `Fix [#1]()`

https://user-images.githubusercontent.com/16184018/113604574-524be400-95fa-11eb-9152-baffbbd4eeb9.mp4

### CODE REVIEW

File: packages/react-inbox/src/components/Messages/styled.ts [hunk 1]
```diff
@@ -46,5 +46,41 @@ export const Empty = styled.div`
...
+  "fontStyle": "normal",
+  "fontWeight": "700",
+  "lineHeight": "14px",
+  "letterSpacing": "0em",
+  "textAlign": "left",
+  "color": "#B9C0CD",
+  "display": "flex",
+  "alignItems": "flex-end",
+  "justifyContent": "flex-end",
+  "> div": {
```
Comment (suhasdeshpande) [inline packages/react-inbox/src/components/Messages/styled.ts:66]: could we perhaps add a more explicit selector here to indicate what this div stands for?

File: packages/react-inbox/src/components/Messages/styled.ts [hunk 2]
```diff
@@ -46,5 +46,41 @@ export const Empty = styled.div`
...
+  height: 4px;
+  flex: 1;
+  opacity: 0.18;
+  :first-child {
+    margin-right: 15px;
+  }
+  :last-child {
+    margin-left: 15px;
+  }
+`;
```
Comment (suhasdeshpande) [inline packages/react-inbox/src/components/Messages/styled.ts:86]: do we need a line break here?

File: packages/react-inbox/src/components/Message/styled.tsx [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import React, { useMemo } from "react";
 import styled from "styled-components";
-import CourierSvg from "./courier-icon.svg";
+import CourierSvg from "../../assets/courier-icon.svg";
```
Comment (rileylnapier) [inline packages/react-inbox/src/components/Message/styled.tsx]: you should be able to do ~/assets

**Review by rileylnapier — APPROVED** [review]

File: packages/react-inbox/src/components/Messages/EndOfTheRoad.tsx [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+function EndOfTheRoad() {
+  return (
+    <div style={{
+      display: "flex",
+      justifyContent: "center",
+      alignItems: "center",
+      flex: "1 0",
+    }}>
+      <Line />
+      <EndPagination /> End Of The Road
```
Comment (suhasdeshpande) [inline packages/react-inbox/src/components/Messages/EndOfTheRoad.tsx:14]: do we see someone calling this `End Of The Road` something else?

Comment (RoyalSix) [reply packages/react-inbox/src/components/Messages/EndOfTheRoad.tsx]: Possibly, added title as prop

File: packages/react-inbox/src/components/Message/styled.tsx (hunk 3 above)
Comment (RoyalSix) [reply packages/react-inbox/src/components/Message/styled.tsx]: There was an issue I encountered with having this and `babel-plugin-root-import`. Fixed.

File: packages/react-inbox/src/components/Messages/styled.ts (hunk 2 above)
Comment (RoyalSix) [reply packages/react-inbox/src/components/Messages/styled.ts:86]: Fixed

File: packages/react-inbox/src/components/Messages/styled.ts (hunk 1 above)
Comment (RoyalSix) [reply packages/react-inbox/src/components/Messages/styled.ts:66]: Fixed.

**Review by rileylnapier — APPROVED** [review]

**Review by suhasdeshpande — APPROVED** [review]

**Pull request merged**